scripts/release/debian/Dockerfile

```diff
@@ -28,8 +28,7 @@ COPY . .
 RUN mkdir -p ./bin/pypi && \
     BUILD_STAGINGDIRECTORY=/azure-cli/bin/pypi ./scripts/release/pypi/build.sh && \
     if [ -d ./privates ]; then find ./privates -name '*.whl' | xargs pip install; fi && \
-    find ./bin/pypi -name '*.whl' | xargs pip install && \
-    pip install --force-reinstall urllib3==1.25.9
+    find ./bin/pypi -name '*.whl' | xargs pip install &&
```

Comment:
This line used to work around https://github.com/Azure/azure-cli/issues/9257, which has already been fixed by latest `requests`.